backend/src/main/java/com/huellapositiva/application/dto/ProfileDto.java

```diff
@@ -0,0 +1,29 @@
+package com.huellapositiva.application.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ProfileDto {
+
+    private final String name;
+    private final String surname;
+    private final String birthDate;
+    private final Integer phoneNumber;
+    private final String email;
+    private final String province;
+    private final String zipCode;
+    private final String town;
+    private final String address;
+    private final String island;
+    private final String photo;
+    private final String curriculumVitae;
+    private final String twitter;
+    private final String instagram;
+    private final String linkedin;
+    private final String additionalInformation;
```

Comment:
TODO Validate

Reply:
It's done, please checked the validation in DTO. Many Thanks.

Reply:
The validation of twitter, instagram and linkedin... Would it be better if we do it in value object of which is a link?

Reply:
As you wish.

Reply:
It's done